client-v2/src/types/Config.ts

```diff
@@ -24,8 +24,7 @@ interface Config {
   sentryPublicDSN: string,
   mediaBaseUrl: string,
   apiBaseUrl: string,
-  stage: string,
-  switches: unknown,
+  switches: {[key: string]: boolean},
```

Comment:
The type here was previously `unknown`, which was breaking store initialisation with state -- this is closer to the type as it stands IRL.